README.md

```diff
@@ -93,6 +98,9 @@ type Options struct {
 	// Please refer to https://pkg.go.dev/golang.org/x/sync/semaphore for more details.
 	NumOfWorkers int64
 
+    	// NameRegexp is used for matching source file name.
+	NameRegexp *regexp.Regexp
```

Comment:
IMO: The rule to know if we, `copy`, should rename the file or not is out of our interest.
It can be regex, datetime, or else.
Thus we should not specifically limit the way to regex.